robmuxinator/robmuxinator.py

```diff
@@ -472,7 +472,7 @@ def __init__(self, ssh_client, session_name, yaml_session, envs=None) -> None:
         command_env_prefix = ""
         if self._envs is not None:
             for env in self._envs:
-                command_env_prefix += "export {}={} && ".format(env[0], env[1])
+                command_env_prefix += "export {}='{}' && ".format(env[0], env[1])
```

Comment:
I think this breaks something for our upstart files, i.e.:
```
PKG_NAV_ENV_CONFIG="'\$(find mojin_default_env_config)'"
```
will no longer be correctly passed to the tmux session...

@Odin-byte 
do you remember the reason why this is important for you and the multi-robot usecase?

Reply:
I added this to correctly parse longer strings including special characters. If there is another solution to this problem, this is no longer needed.

Reply:
But I might just need to inclose longer strings in double quotes like your example to ensure correct parsing.

Reply:
can you give an example for a string that was problematic in you case?

Reply:
I believe it was longer strings like:
```
XAUTHORITY='/run/user/1000/.mutter-Xwaylandauth.I7PAU2'
```

Reply:
But I can image the simpler fix would be to add a pair of double quotes like you did above

Reply:
the problem was rather the `ROSCONSOLE_FORMAT` variable
putting it into `export ROSCONSOLE_FORMAT="'[${severity}] [${time}]: ${message}'"` solved the issue for @Odin-byte

Reply:
@fmessmer it would be good if you could test the correct usage of this variable, because after wrapping it using double and single quotes the error was solved but the logging format was broken. The consoles only showed empty brackets without the severity, time or message.